application/classes/Ushahidi/Repository/Post/Export.php

```diff
@@ -13,13 +13,74 @@
 
 class Ushahidi_Repository_Post_Export extends Ushahidi_Repository_Post
 {
-  public function getFormAttributes($values) {
+
+  public function transformData($data) {
```

Comment:
Am I right that this is basically returning column names for export? Could you rename it to reflect that?